### PR TITLE
Bayes logging

### PR DESCRIPTION
**Pull request by iankronquist**
@willkg r?

### CODE REVIEW

**Comment by iankronquist** [comment]
@willkg r?

File: spicedham/bayes.py [hunk 1]
```diff
@@ -14,6 +17,9 @@ class Bayes(BasePlugin):
 
     def __init__(self, config, backend):
         self.backend = backend
+        self.log = logging.getLogger('spicedham')
+        logging.basicConfig(filename='sh_bayes_classifier.log',
+                            level=logging.DEBUG)
```
Comment (willkg) [inline spicedham/bayes.py]: You don't want to do this. What you want to do is at the module level do this:

```
log = logging.getLogger('spicedham')
```

And then use `log` everywhere. You don't want to do this as an instance member and you don't want to configure logging--whatever is using the library should be in charge of configuring logging.

File: spicedham/bayes.py [hunk 2]
```diff
@@ -62,6 +68,7 @@ def classify(self, response):
             pWordGivenHam = (word['numTotal'] - word['numSpam']) / (total['numTotal'] - total['numSpam'])
             pSpamGivenWord *= (pWordGivenSpam) / pWord
             pHamGivenWord *= pWordGivenHam / pWord
+            self.log.debug(u'word: {0} prob word: {1} prob spam: {2} prob spam: {3}'.format(unicode(description), unicode(pWord), unicode(pWordGivenSpam), unicode(pWordGivenHam), unicode(pWordGivenSpam/pWord), unicode(pWordGivenHam/pWord)))
```
Comment (willkg) [inline spicedham/bayes.py]: Pretty sure this is way over 80 characters.

**Comment by iankronquist** [comment]
@willkg r?

**Comment by willkg** [comment]
Looks good to me! r+

**Pull request merged**